components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -112,33 +113,32 @@ export const PortfolioAsset = (props: Props) => {
   const [nftIframeLoaded, setNftIframeLoaded] = React.useState(false)
   // redux
   const dispatch = useDispatch()
-  const {
-    defaultCurrencies,
-    userVisibleTokensInfo,
-    selectedNetwork,
-    portfolioPriceHistory,
-    selectedPortfolioTimeline,
-    accounts,
-    networkList,
-    transactions,
-    isFetchingPortfolioPriceHistory,
-    transactionSpotPrices,
-    selectedNetworkFilter,
-    coinMarketData,
-    fullTokenList
-  } = useSelector(({ wallet }: { wallet: WalletState }) => wallet)
-
-  const {
-    isFetchingPriceHistory: isLoading,
-    selectedAsset,
-    selectedAssetCryptoPrice,
-    selectedAssetFiatPrice,
-    selectedAssetPriceHistory,
-    selectedTimeline,
-    isFetchingNFTMetadata,
-    nftMetadata,
-    selectedCoinMarket
-  } = useSelector(({ page }: { page: PageState }) => page)
+
+  const defaultCurrencies = useSelector(({ wallet }: { wallet: WalletState }) => wallet.defaultCurrencies)
```

Comment:
Can we update these to use @josheleonard's `safe` and `unsafe` selectors? https://github.com/brave/brave-core/pull/15052

Reply:
@Douglashdaniel Fixed here 7d15e1a43aa3007405158054ef0ada87ded0eaf2